### PR TITLE
Sanem/fsi sco

### PR DESCRIPTION
**Pull request by sasever**
## PR Checklist
* [ ] Closes Issue #xxx
* [ ] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by uday31in** [comment]
Please update [Visio](https://github.com/microsoft/industry/blob/main/fsi/docs/mc4f.vsdx) if not already.

**Review by uday31in — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Microsoft Cloud for Financial Services provides capabilities to manage data to d
 
 - Customer and employee experience
   - [Unified customer profile](./fsi/solutions/unifiedCustomerProfile)
-  - [Customer onboarding](./fsi/solutions/customerOnboarding)
+  - [Customer onboarding](./fsi/solutions/customerOnboarding/readme.md)
```
Comment (marvinbuss) [inline README.md:128]:
```suggestion
  - [Customer onboarding](./fsi/solutions/customerOnboarding)
```

File: fsi/readme.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Overall architecture for Microsoft Cloud for Financial Services.
...
 | Customer and employee experience
-| [Unified customer profile](./fsi/solutions/unifiedCustomerProfile) | Help tailor customer experiences via a comprehensive view of the customer's financial situation
-| [Customer onboarding](./fsi/solutions/customerOnboarding) | Provide customers with easy-access loan apps and self-service tools, streamlining the loan process to help enhance customer experience and loyalty
-| [Collaboration manager](./fsi/solutions/collaborationManager) | Enable automation, collaboration, and communications to help accelerate lending processes, minimize errors, and enhance customer experience
-| [Banking customer engagement](./fsi/solutions/customerEngagement) | Help personalize customer interactions with relevant financial information to accelerate time to resolution
+| [Unified customer profile](./solutions/unifiedCustomerProfile) | Help tailor customer experiences via a comprehensive view of the customer's financial situation
+| [Customer onboarding](./solutions/customerOnboarding/readme.md) | Provide customers with easy-access loan apps and self-service tools, streamlining the loan process to help enhance customer experience and loyalty
```
Comment (marvinbuss) [inline fsi/readme.md:18]:
```suggestion
| [Customer onboarding](./solutions/customerOnboarding) | Provide customers with easy-access loan apps and self-service tools, streamlining the loan process to help enhance customer experience and loyalty
```

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please see my comments. Could you please also provide Design recommendations and considerations as in the other docs?

File: README.md (hunk 1 above)
Comment (sasever) [reply README.md:128]: @marvinbuss Links wasnt working that way that is why I added the readme.md

**Comment by marvinbuss** [comment]
@sasever please resolve conflicts

**Comment by sasever** [comment]
@marvinbuss requested fixes and some additions are done.  can you check?

**Review by marvinbuss — DISMISSED** [review]
LGTM!

**Review by andrehass — APPROVED** [review]
LGTM

**Pull request merged**